package.json

```diff
@@ -25,9 +25,9 @@
     "mocks": "yarn run wiremock --verbose --port 8090 --root-dir test/mocks",
     "test": "node_modules/.bin/jest --testPathPattern=app/bav --coverage",
     "test:browser:report": "node ./test/utils/multiReportGenerator.js",
-    "test:browser": "wait-on tcp:127.0.0.1:8090 tcp:127.0.0.1:5040 && cucumber-js test/browser/features/**/*.feature --require test/browser/support/env.js  --require test/browser/support/setup.js --require test/browser/step_definitions/**/*.js -f json:./test/reports/cucumber_report.json",
+    "test:browser": "wait-on tcp:127.0.0.1:8090 tcp:127.0.0.1:5040 && cucumber-js test/browser/features/**/*.feature --require test/browser/support/env.js  --require test/browser/support/setup.js --require test/browser/step_definitions/**/*.js -f json:./test/reports/cucumber_report.json --tags @success",
```

Comment:
Why do we need the --tags?

Reply:
GitHub actions are still running the test:browser:ci script, so I made sure it only runs the tests tagged with @success and not the new E2E test tagged with @e2e